rootfs/startup.sh

```diff
@@ -43,21 +47,33 @@ cp -r /root/{.gtkrc-2.0,.asoundrc} ${HOME}
 chown -R $USER:$USER $HOME/.[^.]*
 sed -i -e "s|%USER%|$USER|" -e "s|%HOME%|$HOME|" /etc/supervisor/conf.d/supervisord.conf
 
+# check for 'mydata' working directory
+if [ ! -d "$HOME/mydata/" ]; then
```

Comment:
maybe we can find a way to make the app run without any volume mounts

Reply:
added an environmental variable for that